modules/charon-impl/README.md

```diff
@@ -27,7 +27,8 @@ and Group create, get, update, patch and list.
 
 **User create**
 ```
-curl -v --data '{"schemas":[],"name":{"familyName":"jayawardana","givenName":"vindula"},"userName":"vindula","password":"vindula","emails":[{"value":"vindula@work.com","type":"work"}]}' --h```
+curl -v --data '{"schemas":[],"name":{"familyName":"jayawardana","givenName":"vindula"},"userName":"vindula","password":"vindula","emails":[{"value":"vindula@work.com","type":"work"}]}' --h
```

Comment:
This fixes the format error but the command is broken! It should be 
`curl -v --data '{"schemas":[],"name":{"familyName":"jayawardana","givenName":"vindula"},"userName":"vindula","password":"vindula","emails":[{"value":"vindula@work.com","type":"work"}]}' --header "Content-Type:application/scim+json" http://localhost:8080/scim/v2/Users`

Reply:
As you suggested the existing command is wrong.
The command is updated in https://github.com/wso2/charon/pull/234